### ISSUE TITLE
Chains of the Anub'ar in Dragonblight spawns two Tivax

### ISSUE THREAD

**Issue by therinrik** [dalaranwow/dalaran-wow#5497]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
There are three mobs to be killed for items in this quest. One of these characters spawns twice in Icemist Village. They are in the Northernmost structure and the Northeastern-most structure, respectively. All mobs are present; there's just a duplicate.
**Expected behaviour**: Tell us what should happen instead
There should only be one Tivax.
**Steps to reproduce the problem**:

1. Observe that there is indeed a duplicate of Tivax.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Ingame ID of Tivax the Breaker creature / NPC is 26770.
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
Imgur link to four screenshots showing the four NPCs. One should not be there. - https://imgur.com/a/ghEwsZv

**Comment by Golganeth** [comment]
Confirmed. The NPC is supposed to have two possible spawn points but it shouldn't be duplicated.
